server/parsec/client_context.py

```diff
@@ -119,7 +119,7 @@ class AuthenticatedClientContext:
 
     def __post_init__(self):
         self.logger = logger.bind(
-            request=uuid4().hex,
+            request=uuid4().hex[:4],
```

Comment:
Magic number? Maybe add a comment  
```suggestion
            # No need to have the full request uuid in logs
            request=uuid4().hex[:4],
```

Reply:
9a697851672a9249cc9fa6db55d35fa2aa7f3558